rake/tasks/cldr.rake

```diff
@@ -49,6 +49,30 @@ namespace :cldr do
       Worldwide::Cldr::Patch::Patcher.new.perform
     end
 
+    desc <<~DESCRIPTION
+      Generate indices of files in the CLDR data
+
+      This task generates a list of all files in the CLDR data, which is used to speed up the loading of the data.
+
+      eg.: bundle exec rake cldr:data:generate_paths
+    DESCRIPTION
+    task :generate_paths do
```

Comment:
nit: Can you move the body of this task into its own class, following the pattern of the other tasks in this rake file?
```ruby
# rake/cldr/generate_paths.rb
module Worldwide
  module Cldr
    class GeneratePaths
      def perform
        # ...
```
No need to write tests; none of the other rake helpers do.